client/state/notices/middleware.js

```diff
@@ -300,6 +301,26 @@ const onSiteDeleteFailure = ( { error } ) => {
 	return errorNotice( error.message );
 };
 
+const onBillingTransactionRequestFailure = ( { transactionId, error } ) => {
```

Comment:
Some of these have unit tests. Perhaps you'd like to add a couple of tests to cover it?